src/modules/python/pbs_hooks/PBS_power.PY

```diff
@@ -306,8 +306,8 @@ if this_event.type == pbs.PERIODIC:
         try:
             with open(node_file, 'r') as fd:
                 sleep_node_list = fd.read().split(',')
-        except IOError:
-            raise Exception("I/O error reading node file")
+        except IOError, e:
```

Comment:
Please use the more modern style of exception handling (except IOerror as e).  (same for the rest of the exceptions)

Reply:
Fixed. Please have a look.